### ISSUE TITLE
[BUG] Swag Dashboard : No data in Proxies List

### ISSUE THREAD

**Issue by chaletweb** [linuxserver/docker-swag#380]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

No Data in the list of Proxies (Swag Dashboard Mod) even if proxies work.
Before the last release, it worked

### Expected Behavior

list of application should be in the list

### Steps To Reproduce

open the dashboard

### Environment

```markdown
- OS:OpenmediaVault (Debian)
- How docker service was installed:in openmediavault addon
```

### CPU architecture

x86-64

### Docker creation

```bash
docker compose
```

### Container logs

```bash
no data, no error on this point in the logs
```

**Comment by jon-f-novastor** [comment]
I'am having the same issue, there are no proxies listed under the "Proxies" section of SWAG Dashboard UI, I had installed it via docker, and that "Proxies" section WAS listing multiple entires there, but then later that day after restarting containers and such it reverted to showing null entries under the "Proxies" section. I have at least 5 proxy configurations that are active in swag/nginx/proxy-confs/ including SWAG Dashboard (on port 36806).

My docker-compose.yml for swag (latest version) to do with docker-mods is this:

- DOCKER_MODS=linuxserver/mods:swag-dbip|linuxserver/mods:swag-dashboard

My docker-compose.yml for swag (latest version) also contains these "volumes" to mount:
    volumes:
      - /home/jon/appdata/swag:/config
      - /var/log/:/var/log/host:ro
      - /var/lib/fail2ban/fail2ban.sqlite3:/dashboard/fail2ban.sqlite3:ro
      - /var/log/:/dashboard/log:ro

Here are two screenshots of what I'm talking about, taken at the same time on 06/02/2023, with the latest version of swag and the dashboard:

SWAG Dashboard running on the local IP port 81 (showing no Proxies listed):
![image](https://github.com/linuxserver/docker-swag/assets/50121594/8c49172a-7f06-4f94-bf8e-6f26a8243bd9)

SWAG Dashboard running on the nginx proxy-confs port 36806 (showing no Proxies listed):
![image](https://github.com/linuxserver/docker-swag/assets/50121594/96c49b46-eb91-4c94-8d8f-97bedbc4d2d8)

Note: I have tried removing the mods:swag-dbip from swag's docker-compose.yml, and just leaving in the mods:swag-dashboard as my only docker mod entry for swag docker container, but that makes no difference to SWAG Dashboard, it still shows Proxies = null data in the list, when I restart the docker-swag container. I also tried removing the last two "volumes" mentioned above, and it made no difference, to revert to the stock swag docker-compose.yml file. Like I said this was working in the first two hours of setting up swag-dashboard in the latest version of the swag docker container.

docker container inspect xxxcontaineridforswag shows these details:
`jon@betty-pi4:~/docker-swag$ docker container inspect 75f55bd79d43
[
    {
        "Id": "75f55bd79d43d6b4fad5390ee861c5cd19737fb27a924044391a6416f705d300",
        "Created": "2023-06-02T23:55:01.792379838Z",
        "Path": "/init",
        "Args": [],
        "State": {
            "Status": "running",
            "Running": true,
            "Paused": false,
            "Restarting": false,
            "OOMKilled": false,
            "Dead": false,
            "Pid": 424452,
            "ExitCode": 0,
            "Error": "",
            "StartedAt": "2023-06-02T23:55:03.146516062Z",
            "FinishedAt": "0001-01-01T00:00:00Z"
        },
        "Image": "sha256:75cd49ffb2b2a9a09842d9728692015027ea136abd9ca6adf82804852235d429",
        "ResolvConfPath": "/var/lib/docker/containers/75f55bd79d43d6b4fad5390ee861c5cd19737fb27a924044391a6416f705d300/resolv.conf",
        "HostnamePath": "/var/lib/docker/containers/75f55bd79d43d6b4fad5390ee861c5cd19737fb27a924044391a6416f705d300/hostname",
        "HostsPath": "/var/lib/docker/containers/75f55bd79d43d6b4fad5390ee861c5cd19737fb27a924044391a6416f705d300/hosts",
        "LogPath": "/var/lib/docker/containers/75f55bd79d43d6b4fad5390ee861c5cd19737fb27a924044391a6416f705d300/75f55bd79d43d6b4fad5390ee861c5cd19737fb27a924044391a6416f705d300-json.log",
        "Name": "/swag",
        "RestartCount": 0,
        "Driver": "overlay2",
        "Platform": "linux",
        "MountLabel": "",
        "ProcessLabel": "",
        "AppArmorProfile": "docker-default",
        "ExecIDs": null,
        "HostConfig": {
            "Binds": [
                "/var/lib/fail2ban/fail2ban.sqlite3:/dashboard/fail2ban.sqlite3:ro",
                "/var/log:/var/log/host:ro",
                "/var/log:/dashboard/log:ro",
                "/home/jon/appdata/swag:/config:rw"
            ],
            "ContainerIDFile": "",
            "LogConfig": {
                "Type": "json-file",
                "Config": {}
            },
            "NetworkMode": "swag",
            "PortBindings": {
                "36800/tcp": [
                    {
                        "HostIp": "",
                        "HostPort": "36800"
                    }
                ],
                "36801/tcp": [
                    {
                        "HostIp": "",
                        "HostPort": "36801"
                    }
                ],
                "36802/tcp": [
                    {
                        "HostIp": "",
                        "HostPort": "36802"
                    }
                ],
                "36803/tcp": [
                    {
                        "HostIp": "",
                        "HostPort": "36803"
                    }
                ],
                "36804/tcp": [
                    {
                        "HostIp": "",
                        "HostPort": "36804"
                    }
                ],
                "36805/tcp": [
                    {
                        "HostIp": "",
                        "HostPort": "36805"
                    }
                ],
                "36806/tcp": [
                    {
                        "HostIp": "",
                        "HostPort": "36806"
                    }
                ],
                "81/tcp": [
                    {
                        "HostIp": "",
                        "HostPort": "81"
                    }
                ]
            },
            "RestartPolicy": {
                "Name": "always",
                "MaximumRetryCount": 0
            },
            "AutoRemove": false,
            "VolumeDriver": "",
            "VolumesFrom": [],
            "ConsoleSize": [
                0,
                0
            ],
            "CapAdd": [
                "NET_ADMIN"
            ],
            "CapDrop": null,
            "CgroupnsMode": "private",
            "Dns": null,
            "DnsOptions": null,
            "DnsSearch": null,
            "ExtraHosts": null,
            "GroupAdd": null,
            "IpcMode": "private",
            "Cgroup": "",
            "Links": null,
            "OomScoreAdj": 0,
            "PidMode": "",
            "Privileged": false,
            "PublishAllPorts": false,
            "ReadonlyRootfs": false,
            "SecurityOpt": null,
            "UTSMode": "",
            "UsernsMode": "",
            "ShmSize": 67108864,
            "Runtime": "runc",
            "Isolation": "",
            "CpuShares": 0,
            "Memory": 0,
            "NanoCpus": 0,
            "CgroupParent": "",
            "BlkioWeight": 0,
            "BlkioWeightDevice": null,
            "BlkioDeviceReadBps": null,
            "BlkioDeviceWriteBps": null,
            "BlkioDeviceReadIOps": null,
            "BlkioDeviceWriteIOps": null,
            "CpuPeriod": 0,
            "CpuQuota": 0,
            "CpuRealtimePeriod": 0,
            "CpuRealtimeRuntime": 0,
            "CpusetCpus": "",
            "CpusetMems": "",
            "Devices": null,
            "DeviceCgroupRules": null,
            "DeviceRequests": null,
            "MemoryReservation": 0,
            "MemorySwap": 0,
            "MemorySwappiness": null,
            "OomKillDisable": null,
            "PidsLimit": null,
            "Ulimits": null,
            "CpuCount": 0,
            "CpuPercent": 0,
            "IOMaximumIOps": 0,
            "IOMaximumBandwidth": 0,
            "MaskedPaths": [
                "/proc/asound",
                "/proc/acpi",
                "/proc/kcore",
                "/proc/keys",
                "/proc/latency_stats",
                "/proc/timer_list",
                "/proc/timer_stats",
                "/proc/sched_debug",
                "/proc/scsi",
                "/sys/firmware"
            ],
            "ReadonlyPaths": [
                "/proc/bus",
                "/proc/fs",
                "/proc/irq",
                "/proc/sys",
                "/proc/sysrq-trigger"
            ]
        },
        "GraphDriver": {
            "Data": {
                "LowerDir": "/var/lib/docker/overlay2/20da244be2976cde5c1aadca812395b4e1e77874adc669caf18d712a68ab50b1-init/diff:/var/lib/docker/overlay2/92ffd2c712f18d81358934adbaeb57f578372dc52602a4fcce486ff11012ac85/diff:/var/lib/docker/overlay2/7bbef4f27fcd47b9d34f7cf3a009cb63cd1949f504ac4b99240fa7b63506dfc3/diff:/var/lib/docker/overlay2/35d96fd8b9e5d3352bccbed498419206f2f1ce925805c039a560162e934c4de2/diff:/var/lib/docker/overlay2/ff05b0cdad95f29c25c7cfaf6ea2b237bb901e7a09541d28e173f87d9d9bfad6/diff:/var/lib/docker/overlay2/b58cd664c5185bd97e7d5473b3c0b3fe64bd99d5472d23c391c6b257a5f31fab/diff:/var/lib/docker/overlay2/71bcbd71863b3071b616a91c8c5e0a899db641966bd441ad8d346a1064f6e5b1/diff:/var/lib/docker/overlay2/86935dc8086357ca4d904a4ae9abfaccfecb06114fcfc817c2ca3abc286bd0af/diff:/var/lib/docker/overlay2/3a31d0d11a038a259dcbe7305136a4a73e67edd2020ab21cb83031dfe4d3f278/diff:/var/lib/docker/overlay2/82d536b08d7a4836b6cd463c95766dd09de4003f4a9b1b860bc5895927c75879/diff",
                "MergedDir": "/var/lib/docker/overlay2/20da244be2976cde5c1aadca812395b4e1e77874adc669caf18d712a68ab50b1/merged",
                "UpperDir": "/var/lib/docker/overlay2/20da244be2976cde5c1aadca812395b4e1e77874adc669caf18d712a68ab50b1/diff",
                "WorkDir": "/var/lib/docker/overlay2/20da244be2976cde5c1aadca812395b4e1e77874adc669caf18d712a68ab50b1/work"
            },
            "Name": "overlay2"
        },
        "Mounts": [
            {
                "Type": "bind",
                "Source": "/var/lib/fail2ban/fail2ban.sqlite3",
                "Destination": "/dashboard/fail2ban.sqlite3",
                "Mode": "ro",
                "RW": false,
                "Propagation": "rprivate"
            },
            {
                "Type": "bind",
                "Source": "/var/log",
                "Destination": "/var/log/host",
                "Mode": "ro",
                "RW": false,
                "Propagation": "rprivate"
            },
            {
                "Type": "bind",
                "Source": "/var/log",
                "Destination": "/dashboard/log",
                "Mode": "ro",
                "RW": false,
                "Propagation": "rprivate"
            },
            {
                "Type": "bind",
                "Source": "/home/jon/appdata/swag",
                "Destination": "/config",
                "Mode": "rw",
                "RW": true,
                "Propagation": "rprivate"
            }
        ],
        "Config": {
            "Hostname": "75f55bd79d43",
            "Domainname": "",
            "User": "",
            "AttachStdin": false,
            "AttachStdout": false,
            "AttachStderr": false,
            "ExposedPorts": {
                "36800/tcp": {},
                "36801/tcp": {},
                "36802/tcp": {},
                "36803/tcp": {},
                "36804/tcp": {},
                "36805/tcp": {},
                "36806/tcp": {},
                "443/tcp": {},
                "80/tcp": {},
                "81/tcp": {}
            },
            "Tty": false,
            "OpenStdin": false,
            "StdinOnce": false,
            "Env": [
                "PUID=1000",
                "PGID=1000",
                "TZ=America/Los_Angeles",
                "URL=x.ddns.net",
                "VALIDATION=http",
                "SUBDOMAINS=",
                "CERTPROVIDER=",
                "DNSPLUGIN=",
                "PROPAGATION=",
                "EMAIL=qsnexus@gmail.com",
                "ONLY_SUBDOMAINS=false",
                "STAGING=false",
                "DOCKER_MODS=linuxserver/mods:swag-dbip|linuxserver/mods:swag-dashboard",
                "PATH=/lsiopy/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin",
                "PS1=$(whoami)@$(hostname):$(pwd)\\$ ",
                "HOME=/root",
                "TERM=xterm",
                "S6_CMD_WAIT_FOR_SERVICES_MAXTIME=0",
                "S6_VERBOSITY=1",
                "S6_STAGE2_HOOK=/docker-mods",
                "VIRTUAL_ENV=/lsiopy",
                "LSIO_FIRST_PARTY=true",
                "DHLEVEL=2048",
                "AWS_CONFIG_FILE=/config/dns-conf/route53.ini",
                "S6_BEHAVIOUR_IF_STAGE2_FAILS=2"
            ],
            "Cmd": null,
            "Image": "lscr.io/linuxserver/swag:latest",
            "Volumes": {
                "/config": {},
                "/dashboard/fail2ban.sqlite3": {},
                "/dashboard/log": {},
                "/var/log/host": {}
            },
            "WorkingDir": "/",
            "Entrypoint": [
                "/init"
            ],
            "OnBuild": null,
            "Labels": {
                "build_version": "Linuxserver.io version:- 2.6.0-ls214 Build-date:- 2023-05-31T16:33:39+00:00",
                "com.docker.compose.config-hash": "1ea368558cca45ce13901367c7dda29dd78c6cc9687fca6bbacc52d3742e2836",
                "com.docker.compose.container-number": "1",
                "com.docker.compose.oneoff": "False",
                "com.docker.compose.project": "docker-swag",
                "com.docker.compose.project.config_files": "docker-compose.yml",
                "com.docker.compose.project.working_dir": "/home/jon/docker-swag",
                "com.docker.compose.service": "swag",
                "com.docker.compose.version": "1.29.2",
                "maintainer": "nemchik",
                "org.opencontainers.image.authors": "linuxserver.io",
                "org.opencontainers.image.created": "2023-05-31T16:33:39+00:00",
                "org.opencontainers.image.description": "SWAG - Secure Web Application Gateway (formerly known as letsencrypt, no relation to Let's Encrypt™) sets up an Nginx webserver and reverse proxy with php support and a built-in certbot client that automates free SSL server certificate generation and renewal processes (Let's Encrypt and ZeroSSL). It also contains fail2ban for intrusion prevention.",
                "org.opencontainers.image.documentation": "https://docs.linuxserver.io/images/docker-swag",
                "org.opencontainers.image.licenses": "GPL-3.0-only",
                "org.opencontainers.image.ref.name": "60f6fafc535be4849376a593d4f6d4cb9e6acbb9",
                "org.opencontainers.image.revision": "60f6fafc535be4849376a593d4f6d4cb9e6acbb9",
                "org.opencontainers.image.source": "https://github.com/linuxserver/docker-swag",
                "org.opencontainers.image.title": "Swag",
                "org.opencontainers.image.url": "https://github.com/linuxserver/docker-swag/packages",
                "org.opencontainers.image.vendor": "linuxserver.io",
                "org.opencontainers.image.version": "2.6.0-ls214"
            }
        },
        "NetworkSettings": {
            "Bridge": "",
            "SandboxID": "a94f9ce65bdc8b54febf12d9d1ea7fa543336071f5859aed1cc3a9c2058d83fe",
            "HairpinMode": false,
            "LinkLocalIPv6Address": "",
            "LinkLocalIPv6PrefixLen": 0,
            "Ports": {
                "36800/tcp": [
                    {
                        "HostIp": "0.0.0.0",
                        "HostPort": "36800"
                    },
                    {
                        "HostIp": "::",
                        "HostPort": "36800"
                    }
                ],
                "36801/tcp": [
                    {
                        "HostIp": "0.0.0.0",
                        "HostPort": "36801"
                    },
                    {
                        "HostIp": "::",
                        "HostPort": "36801"
                    }
                ],
                "36802/tcp": [
                    {
                        "HostIp": "0.0.0.0",
                        "HostPort": "36802"
                    },
                    {
                        "HostIp": "::",
                        "HostPort": "36802"
                    }
                ],
                "36803/tcp": [
                    {
                        "HostIp": "0.0.0.0",
                        "HostPort": "36803"
                    },
                    {
                        "HostIp": "::",
                        "HostPort": "36803"
                    }
                ],
                "36804/tcp": [
                    {
                        "HostIp": "0.0.0.0",
                        "HostPort": "36804"
                    },
                    {
                        "HostIp": "::",
                        "HostPort": "36804"
                    }
                ],
                "36805/tcp": [
                    {
                        "HostIp": "0.0.0.0",
                        "HostPort": "36805"
                    },
                    {
                        "HostIp": "::",
                        "HostPort": "36805"
                    }
                ],
                "36806/tcp": [
                    {
                        "HostIp": "0.0.0.0",
                        "HostPort": "36806"
                    },
                    {
                        "HostIp": "::",
                        "HostPort": "36806"
                    }
                ],
                "443/tcp": null,
                "80/tcp": null,
                "81/tcp": [
                    {
                        "HostIp": "0.0.0.0",
                        "HostPort": "81"
                    },
                    {
                        "HostIp": "::",
                        "HostPort": "81"
                    }
                ]
            },
            "SandboxKey": "/var/run/docker/netns/a94f9ce65bdc",
            "SecondaryIPAddresses": null,
            "SecondaryIPv6Addresses": null,
            "EndpointID": "",
            "Gateway": "",
            "GlobalIPv6Address": "",
            "GlobalIPv6PrefixLen": 0,
            "IPAddress": "",
            "IPPrefixLen": 0,
            "IPv6Gateway": "",
            "MacAddress": "",
            "Networks": {
                "swag": {
                    "IPAMConfig": null,
                    "Links": null,
                    "Aliases": [
                        "75f55bd79d43",
                        "swag"
                    ],
                    "NetworkID": "b903e5d8bdc2b2ab347efedb9fe81b6b21ff352a5986770be51fcdc6c1d146c7",
                    "EndpointID": "0ff3c93e99ecc586642f98cae3bcea90d960fe006f267daa6043f85ba3a4d1d6",
                    "Gateway": "192.168.16.1",
                    "IPAddress": "192.168.16.9",
                    "IPPrefixLen": 20,
                    "IPv6Gateway": "",
                    "GlobalIPv6Address": "",
                    "GlobalIPv6PrefixLen": 0,
                    "MacAddress": "02:42:c0:a8:10:09",
                    "DriverOpts": null
                }
            }
        }
    }
]
`

**Comment by jon-f-novastor** [comment]
> Thanks for opening your first issue here! Be sure to follow the relevant issue templates, or risk having this issue marked as invalid.

I'm another person who has hit this issue, whereby the "Proxies" section did list multiple proxies in the first few hours after installing the mod for the first time on 05/27/2023, but then a few hours I noticed there were no proxies being listed under the "Proxies" section of the UI. I have added my details and I hope that helps, as I do think that it is true that there is a bug causing this. I believe this is a bug in the "mods:swag-dashboard" + swag "lscr.io/linuxserver/swag:latest" docker app, with the swag-dashboard mod added, so hopefully my added details in the one prior comment above this help you to track down the issue.
